modules/node/src/config/config.service.ts

```diff
@@ -120,6 +120,7 @@ export class ConfigService implements OnModuleInit {
 
     if (currentChainId !== 1) {
       const tokenConfig = await this.getTestnetTokenConfig();
+      console.log('tokenConfig: ', tokenConfig);
```

Comment:
`console.log` are great for debugging but let's either remove them or replace w proper logging system before merging